src/main/java/com/jagrosh/jmusicbot/settings/SettingsManager.java

```diff
@@ -58,6 +58,8 @@ public SettingsManager()
                         o.has("skip_ratio")      ? o.getDouble("skip_ratio")                 : SKIP_RATIO));
             });
         } catch(IOException | JSONException e) {
+            if (System.getProperty("serversettings.suppressFirstLoadWarning", "false").equalsIgnoreCase("true"))
+                return;
```

Comment:
This seems to be out of scope of this pull request. Personally, I would prefer if this change were to be made in a new pull request.